code/game/gamemodes/cult/cult_items.dm

```diff
@@ -682,3 +682,107 @@
 	desc = "It's an orb of crystalized blood. Can be used to transfer blood between cultists."
 	var/blood = 50
 
+/obj/item/portal_amulet
+	name = "reality sunderer"
+	icon = 'icons/obj/cult.dmi'
+	icon_state = "amulet"
+	desc = "Some amulet made out of metal, bluespace crystals, and blood. Allows cultists to open portals over teleport runes, destroying the rune in the process."
+	w_class = WEIGHT_CLASS_SMALL
+
+
+/obj/item/portal_amulet/afterattack(atom/O, mob/user, proximity)
+	. = ..()
+	if(!iscultist(user) && ishuman(user))
+		var/mob/living/carbon/human/H = user
+		to_chat(H, "<span class='cultlarge'>\"So, you want to explore space?\"</span>")
+		to_chat(H, "<span class='warning'>Space flashes around you as you are moved somewhere else!</span>")
+		H.Confused(10)
+		H.flash_eyes(override_blindness_check = TRUE)
+		H.EyeBlind(10)
+		do_teleport(H, get_turf(H), 5, asoundin = 'sound/magic/cult_spell.ogg')
+		qdel(src)
+		return
+
+	if(istype(O, /obj/effect/rune))
+		if(!istype(O, /obj/effect/rune/teleport))
+			to_chat(user, "<span class='warning'>[src] only works on teleport runes.</span>")
+			return
+		if(!proximity)
+			to_chat(user, "<span class='warning'>You are too far away from the teleport rune.</span>")
+			return
+		var/obj/effect/rune/teleport/R = O
+		attempt_portal(R, user)
+
+/obj/item/portal_amulet/proc/attempt_portal(obj/effect/rune/teleport/R, mob/user)
+	var/list/potential_runes = list()
+	var/list/teleportnames = list()
+	var/list/duplicaterunecount = list()
+	var/turf/T = get_turf(src) //used to tell the other rune where we came from
+
+	for(var/I in GLOB.teleport_runes)
+		var/obj/effect/rune/teleport/target = I
+		var/resultkey = target.listkey
+		if(resultkey in teleportnames)
+			duplicaterunecount[resultkey]++
+			resultkey = "[resultkey] ([duplicaterunecount[resultkey]])"
+		else
+			teleportnames += resultkey
+			duplicaterunecount[resultkey] = 1
+		if(target != R && is_level_reachable(target.z))
```

Comment:
Might want to do this check before the rest. Since if this isn't valid then the rune won't be shown in the list anyway

Reply:
done